testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -412,6 +412,84 @@ QUnit.test('From renders the right types of editors according to stylingMode opt
     assert.ok($testContainer.find('.dx-field-item .dx-textbox').hasClass('dx-editor-underlined'), 'right class rendered');
 });
 
+QUnit.module('Checkbox editor value', () => {
+    function checkCheckbox(editor, newBoolValue) {
+        const expectedValue = newBoolValue === 'not set' ? false : newBoolValue;
+        QUnit.assert.equal(editor.option('value'), expectedValue, `editor has ${expectedValue} value`);
```

Comment:
I think we should do this check as well to avoid unexpected changes in the passed formData object:

`assert.equal(form.option('formData').boolMember, expectedValue, `form.option('formData').boolMember`);`

Reply:
No, it is the jquery wrapper method.
I have use id to check the css class of the checkbox